### DISCUSSION TITLE
Key reconfiguration?

### DISCUSSION THREAD

**Discussion by camji** [nirgoren/Eddienput#9] [Ideas]
I tried this on IKEMEN and quickly found out that the current hot keys overlap with the debug buttons in IKEMEN, It would be neat if you could reconfigure the keys  for eddienput

**Comment by DesChamos** [comment]
If you want a temporary fix, it looks like this is handled in src/gui.py under def on_press(key). It would be nice to have this built into the ui in some way in the future.

**Reply by KuukuuOmoi** [reply]
wait where is this file? does it also handle the syntaxes for "boop"

**Comment by nirgoren** [comment]
A way to allow users to reconfigure the hotkeys will be added in a future release.
